edx_rest_framework_extensions/permissions.py

```diff
@@ -144,6 +144,26 @@ def _get_user_filter(self, request):
         return None
 
 
+class LoginRequired(IsAuthenticated):
+    """
+    A non-standard DRF permission class that will login redirect unauthorized users.
+
+    It can be used to convert a plan Django view using @login_required into a
+    DRF APIView, which is useful to enable our DRF JwtAuthentication class.
+
+    This permission is non-standard because DRF only supports returning a 401
```

Comment:
Once again, not sure how `non-standard` this is.  The `@login_required` decorator redirected to login as well.

Reply:
See other comment.

Reply:
Fixed.